test/ui-test/testSuites/suite_onboarding/tst_signUpNegativeCases/test.feature

```diff
@@ -10,77 +10,43 @@ Feature: Status Desktop Sign Up, negative cases
 
     Examples:
-      | username |
-      | Athl     |
-      | Gra      |
-      | tester3@ |
-
-  Scenario Outline: The user cannot sign up with wrong password format in both new password and confirmation input
-   Given the user inputs username "<username>"
-    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_newPsw_Input"
-    And the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPsw_Input"
-    Then the following ui-component "onboarding_create_password_button" is not enabled
-
-    Examples:
-      | username  | wrongpassword |
-      | tester124 | badP          |
-
-  Scenario Outline: The user cannot sign up with right password format in new password input but incorrect in confirmation password input
-    Given the user inputs username "<username>"
-    And the user inputs the following "<password>" with ui-component "onboarding_newPsw_Input"
-    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPsw_Input"
-    Then the following ui-component "onboarding_create_password_button" is not enabled
-
-    Examples:
-      | username  | wrongpassword | password         |
-      | tester124 | bad2!s        | TesTEr16843/!@01 |
-
-  Scenario Outline: The user cannot sign up with incorrect confirmation-again password
-    Given the user inputs username "<username>"
-    And the user inputs the following "<password>" with ui-component "onboarding_newPsw_Input"
-    And the user inputs the following "<password>" with ui-component "onboarding_confirmPsw_Input"
-    And the user clicks on the following ui-component "onboarding_create_password_button"
-    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPswAgain_Input"
-    Then the following ui-component "onboarding_finalise_password_button" is not enabled
+      | username | reason       |
+      |          | empty        |
+      | 1234     | 1 char short |
+      | 1234!    | invalid char |
+
+
+  Scenario Outline: <reason> password creation
+  Given the user inputs username "tester123"
+   When "<password-1>" is entered into the first password field
+   And "<password-2>" is entered into the second password field
+   Then the create password button is disabled
+
+   Examples:
+   | password-1       | password-2             | button-state | reason                     |
+   |                  |                        | disabled     | empty                      |
+   | 1234567890       |                        | disabled     | valid/empty                |
+   |                  | 1234567890             | disabled     | empty/valid                |
+   | 123456789        | 123456789              | disabled     | 1 char short               |
+   | TesTEr16843/!@01 | 1234567890             | disabled     | mismatched                 |
+  # | TesTEr16843/!@01 | TesTEr16843/!@01       | enabled      | matching valid passwords   |
```

Comment:
Why not added?

Reply:
The scenario on line 39 is a valid/positive scenario. The feature file appears to contain only negative cases so I thought the positive one should be covered elsewhere.